### PR TITLE
https://github.com/praxisdigital/moodle-opgaver/issues/504 - Fixed JS…

### PR DESCRIPTION
**Pull request by AraPraxis**
… and updated curl

### CODE REVIEW

File: classes/utils.php [hunk 1]
```diff
@@ -51,32 +51,37 @@ public static function tinymce_planetestream_obfuscate($strx) {
...
+
+            $curl = new \curl();
+            $curl->setopt([
+                'CONNECTTIMEOUT' => 10,
+                'TIMEOUT' => 10,
+                'RETURNTRANSFER' => true,
+                'FOLLOWLOCATION' => true,
+                'MAXREDIRS' => 4,
+                'FORBID_REUSE' => true,
+                'SSL_VERIFYPEER' => false
```
Comment (frederikmillingpytlick) [inline classes/utils.php:67]: Why not use CURLOPT_xxx constants?

File: classes/utils.php [hunk 2]
```diff
@@ -51,32 +51,37 @@ public static function tinymce_planetestream_obfuscate($strx) {
...
-}
+    public static function tinymce_planetestream_getauthticket($url, $checksum, $delta, $userip, &$params) {
+        $return = '';
+        try {
+            $url .= '/VLE/Moodle/Auth/?source=1&checksum=' . $checksum . '&delta=' . $delta . '&u=' . $userip;
+
+            $curl = new \curl();
+            $curl->setopt([
+                'CONNECTTIMEOUT' => 10,
+                'TIMEOUT' => 10,
```
Comment (frederikmillingpytlick) [inline classes/utils.php:62]: Still a pretty long timeout, but maybe it's okay

**Review by frederikmillingpytlick — APPROVED** [review]

File: classes/utils.php (hunk 1 above)
Comment (AraPraxis) [reply classes/utils.php:67]: Because Moodle :)
image

File: classes/utils.php (hunk 2 above)
Comment (AraPraxis) [reply classes/utils.php:62]: :crossed_fingers:

**Pull request merged**